spec/workers/sidekiq_mail_worker_spec.rb

```diff
@@ -7,21 +7,32 @@
   before(:each) do
     allow(Assignment).to receive(:find).with('1').and_return(assignment)
     allow(Participant).to receive(:where).with(parent_id: '1').and_return([participant])
+    allow(User).to receive(:where).with(email: "psingh22@ncsu.edu").and_return([user])
+    allow(Participant).to receive(:find).with(user_id: '1', parent_id: '1').and_return([participant])
```

Comment:
i dont think that find works this way. find only accepts id. this should be where i think